synapse/lib/aha.py

```diff
@@ -177,6 +177,23 @@ async def modAhaSvcInfo(self, name, svcinfo):
 
         return await self.cell.modAhaSvcInfo(name, svcinfo)
 
+    async def getAhaSvcMirrors(self, name):
+        '''
+        Return list of AHA svcinfo dictionaries for mirrors of a service.
+        '''
+        svcinfo = await self.cell.getAhaSvc(name)
+        if svcinfo is None:
+            return None
+
+        svcnetw = svcinfo.get('svcnetw')
+        await self._reqUserAllowed(('aha', 'service', 'get', svcnetw))
+
+        svciden = svcinfo['svcinfo']['iden']
+
+        mirrors = await self.cell.getAhaSvcMirrors(svciden)
+
+        return s_msgpack.deepcopy(mirrors)
```

Comment:
Since this is the telepath API, ideally this should end up going to daemon.py/t2call, then link.py/tx and msgpacked. Do we need to do a deepcopy here?

Reply:
removed (this was leftover from when I was originally going to mutate before returning)